### PR TITLE
Fixing Bahasa Indonesa translation

### PR DESCRIPTION
**Pull request by rizafahmi**
What's changed, or what was fixed?
- Fixing Bahasa Indonesia translation generated by machine

- [ ] This has been reviewed and approved by (NAME)
- [x] I have run `npm test` locally and all tests pass.
- [ ] I have added the appropriate `type-something` label.
- [x] I've staged the site and manually verified that my content displays correctly.

**CC:** @petele

### CODE REVIEW

**Comment by rizafahmi** [comment]
I signed it!

File: src/content/id/fundamentals/codelabs/your-first-pwapp/index.md [hunk 1]
```diff
@@ -77,37 +77,37 @@ Untuk menguji apakah Kunci API Anda berfungsi dengan benar, buat permintaan HTTP
...
-4. Klon __https: //github.com/googlecodelabs/your-first-pwapp.git__ dan klik OK.
+4. Duplikasi (_clone_) __https: //github.com/googlecodelabs/your-first-pwapp.git__ dan klik OK.
 5. Setelah repo dimuat, edit file `.env` , dan perbarui dengan kunci API DarkSky Anda. 6. Klik tombol __Tampilkan Live__ untuk melihat PWA dalam aksi.
 
 #### Alternatif: Unduh kode &amp; bekerja secara lokal
 
 Jika Anda ingin mengunduh kode dan bekerja secara lokal, Anda harus memiliki Node versi terbaru, dan pengaturan editor kode dan siap untuk digunakan.
 
-Caution: Jika Anda bekerja secara lokal, beberapa audit Lighthouse tidak akan berlalu, dan instalasi mungkin tidak tersedia karena server lokal tidak menyajikan konten dalam konteks yang aman.
+Perhatian: Jika Anda bekerja secara lokal, beberapa audit Lighthouse tidak akan terpenuhi, dan instalasi mungkin tidak tersedia karena server lokal tidak menyajikan konten dalam konteks yang aman.
```
Comment (petele) [inline src/content/id/fundamentals/codelabs/your-first-pwapp/index.md]: Please leave lines starting with "Caution:" untranslated, it's a special keyword that does some extra formatting. The same goes for "Note:", "Warning:", "Success:", "Key Point:", "Key Term:", and "Objective:"

Would you update throughout the article please?

File: src/content/id/fundamentals/codelabs/your-first-pwapp/index.md [hunk 2]
```diff
@@ -15,59 +15,59 @@ book_path: /web/fundamentals/_book.yaml
 
 ### Apa yang membuat aplikasi web, Aplikasi Web Progresif?
 
-Aplikasi Web Progresif menyediakan pengalaman yang dapat diinstal dan seperti aplikasi di desktop dan seluler yang dibangun dan dikirim langsung melalui web. Itu adalah aplikasi web yang cepat dan andal. Dan yang paling penting, mereka adalah aplikasi web yang berfungsi di browser apa pun. Jika Anda membuat aplikasi web hari ini, Anda sudah berada di jalur untuk membangun Aplikasi Web Progresif.
+Aplikasi Web Progresif menyediakan pengalaman yang dapat diinstal dan seperti aplikasi di desktop dan seluler yang dibangun dan dikirim langsung melalui web. Itu adalah aplikasi web yang cepat dan andal. Dan yang paling penting, aplikasi tersebut adalah aplikasi web yang berjalan di browser apa pun. Jika Anda membuat aplikasi web hari ini, Anda sudah berada di jalur untuk membangun Aplikasi Web Progresif.
```
Comment (ivankristianto) [inline src/content/id/fundamentals/codelabs/your-first-pwapp/index.md]:
```suggestion
Aplikasi Web Progresif menyediakan pengalaman yang dapat diinstal dan seperti aplikasi di __desktop__ dan seluler yang dibangun dan dikirim langsung melalui web. Itu adalah aplikasi web yang cepat dan andal. Dan yang paling penting, aplikasi tersebut adalah aplikasi web yang berjalan di pramban apa pun. Jika Anda membuat aplikasi web hari ini, Anda sudah berada di jalur untuk membangun Aplikasi Web Progresif.
```

File: src/content/id/fundamentals/codelabs/your-first-pwapp/index.md (hunk 1 above)
Comment (rizafahmi) [reply src/content/id/fundamentals/codelabs/your-first-pwapp/index.md]: Yes, I will

**Pull request merged**